src/platform/PICO/uart/uart_rx.pio.h

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+// -------------------------------------------------- //
+// This file is autogenerated by pioasm; do not edit! //
+// -------------------------------------------------- //
+
+#pragma once
+
+#if !PICO_NO_HARDWARE
+#include "hardware/pio.h"
+#endif
+
+// ------------ //
+// uart_rx_mini //
+// ------------ //
+
+#define uart_rx_mini_wrap_target 0
+#define uart_rx_mini_wrap 3
+#define uart_rx_mini_pio_version 0
+
+static const uint16_t uart_rx_mini_program_instructions[] = {
```

Comment:
This is in header file - creating separate instance of  uart_rx_mini_program_instructions for each file it is included in. This should probably be .c file